packages/orchestration/src/proposals/start-stakeOsmo.js

```diff
@@ -1,7 +1,7 @@
 import { makeTracer } from '@agoric/internal';
 import { makeStorageNodeChild } from '@agoric/internal/src/lib-chainStorage.js';
 import { E } from '@endo/far';
-import { makeChainHub } from '../utils/chainHub.js';
+import { getChainsAndConnection, makeChainHub } from '../utils/chainHub.js';
```

Comment:
Thinking on this more, wrt to host code requirements for async-flow, it may make more sense to have this directly on `makeChainHub` instead of a helper. NB for this PR but something to consider.